includes/emails/actions.php

```diff
@@ -28,6 +28,12 @@ function edd_trigger_purchase_receipt( $payment_id = 0, $payment = null, $custom
 	if ( isset( $_POST['edd-action'] ) && 'edit_payment' == $_POST['edd-action'] ) {
 		return;
 	}
+	if ( null === $payment ) {
+		$payment = new EDD_Payment( $payment_id );
+	}
+	if ( 'refund' === $payment->order->type ) {
```

Comment:
```suggestion
	if ( $payment->order instanceof \EDD\Orders\Order && 'refund' === $payment->order->type ) {
```

In the unlikely event that `$payment->order` ended up `false`.